README.md

```diff
@@ -29,6 +30,7 @@ Hosted payment processors run on someone else's server. This simplifies the init
 
 | Processor | Fees | Lightning | Directly to Your Wallet | Conversion to Fiat | Requirements |
 | --------- |:----:|:---------:|:-----------------------:|:------------------:| ------------ |
+| [Apirone](https://www.apirone.com) | Fix 0.0002 BTC | No | Yes | Via [Payment Forwarding](https://apirone.com/docs/crypto-forwarding-api) | No |
```

Comment:
- Move to the bottom of the list
- Update "Fix 0.0002 BTC" -> "0.0002 BTC" for consistency